app/views/home/index.html.slim

```diff
@@ -12,35 +12,29 @@
         .content-wrapper
           .content-body
             section.content.landing-part2
-              h2 Our proposition
+              h2 OUR BENEFITS
```

Comment:
add `text-transform` to upper case for such things, there's no need to write capital letters in HTML